### PR TITLE
Added card component

### PR DESCRIPTION
**Pull request by zoef-yt**

### CODE REVIEW

File: css/main.css [hunk 1]
```diff
@@ -24,6 +24,7 @@
 	--colour-warning-third: #624711;
 	--colour-error: #f8513d;
 	--colour-error-second: #6b2a2b;
+	--colour-black-with-opacity: rgba(0, 0, 0, 0.5);
```
Comment (sanu29) [inline css/main.css]: instead of black dark word could be used and with could be omited

**Review by sanu29 — APPROVED** [review]
All good. naming consistency could be worked on as well class names could be better.
Rest all good. Spacing and code is readable.

File: css/main.css (hunk 1 above)
Comment (zoef-yt) [reply css/main.css]: sure will make those changes

**Review by amartya-rs — APPROVED** [review]
Just a suggestion, instead of using svg directly in html file you can use img tags and provide the link of the svg in it.
Rest looks good to me.

**Review by archit7448 — APPROVED** [review]
Amazing work good to merge

**Comment by zoef-yt** [comment]
> Just a suggestion, instead of using svg directly in html file you can use img tags and provide the link of the svg in it. Rest looks good to me.

yeah the HTML is not going to be part of the end projects its just for visual debugging. should merge?

**Comment by amartya-rs** [comment]
> > Just a suggestion, instead of using svg directly in html file you can use img tags and provide the link of the svg in it. Rest looks good to me.
> 
> yeah the HTML is not going to be part of the end projects its just for visual debugging. should merge?

Okay, then you can merge.

**Review by tonymike99 — APPROVED** [review]
Good to go!

**Pull request merged**